src/main/webapp/app/entities/participation/participation.service.ts

```diff
@@ -165,4 +168,27 @@ export class ParticipationService {
         }
         return convertedSubmissions;
     }
+
+    mergeParticipations(participations: Participation[]): Participation {
```

Comment:
I think the naming could be better, it is more mergeResultsAndSubmissions (as a suggestion).

Reply:
good point. i renamed it

Reply:
Cool!